firmware/robot2015/src-kckr/kicker.h

```diff
@@ -0,0 +1,4 @@
+#pragma once
```

Comment:
It may be slightly better to just put the includes in the files that need them and get rid of kicker.h.